### PR TITLE
startprefix: add instructions to leave prefix

### PR DESCRIPTION
**Pull request by palao**
When the prefix is started, write a message to tell the user how to "deactivate" the prefix.

### CODE REVIEW

**Comment by thesamesam** [comment]
@grobian - look ok?

**Comment by grobian** [comment]
I find it a bit over-information.  But if considered that useful, I'd reword it in something more generic, like
`exit this shell to leave Gentoo Prefix again`.

Because `^D` works, and non-bash shells might even respond to other commands than `exit`.

Perhaps to avoid the extra line of info, we could change the "Entering" message into something that indicates a shell is launched?  Like "Starting Gentoo Prefix shell for ${EPREFIX}".

Thoughts?

**Comment by palao** [comment]
It is hard for me to judge how much is too much information. But I think some information is needed. 

I found this very nice project a couple of weeks ago, and after starting the prefix I felt confused when I wanted to finish. I remember thinking "Is `startprefix` just setting some variables, or is it a new shell?"
I had to inspect the script to learn that it was a shell and then I knew that `exit` was the command.

I thought this could be improved very easily with a little message, hence the PR.

I'm not sure of the behavior of the prefix on all systems that could make use of it (for instance, what happens on Cygwin, or on exotic shells?). From this point of view, your message is better because it is certainly accurate.

On the other hand,  `type 'exit' to leave` is a more concrete statement. And if you know a bit about the shell, you will also know that `^D` will probably work.
BTW,  `exit` will work in the most known shells (Bash, Zsh, Ksh, Fish, dash, at least).

Concerning your last suggestion: in my opinion, an explicit message is better than an implicit one.

These are my thoughts.

Thank you for this great project.

**Comment by grobian** [comment]
How about `Starting Gentoo Prefix shell ${SHELL}`?

**Comment by palao** [comment]
In my opinion it is better than how it is now. Still a bit too implicit for my taste.
 Would it be acceptable something like
`Starting Gentoo Prefix shell ${SHELL}. To deactivate the prefix, exit the shell`?

**Comment by grobian** [comment]
Ok, I'm not convinced, but I see your point.  Since a typical EPREFIX will make your last suggestion exceed an ordinary terminal width, I went for your first multi-line suggestion, but used different wording.

**Comment by grobian** [comment]
The only way to do it in a single line would probably be without mentioning the EPREFIX:
`Activating Gentoo Prefix, exit the shell to deactivate again`

**Comment by palao** [comment]
> Ok, I'm not convinced, but I see your point. Since a typical EPREFIX will make your last suggestion exceed an ordinary terminal width, I went for your first multi-line suggestion, but used different wording.

Ah, I understand now some of your concerns. I didn't think that the message should fit into a line of 80 (?) chars.
Anyway, thank you for taking into account my suggestion (this PR).
